data-serving/data-service/Dockerfile

```diff
@@ -23,5 +26,25 @@ RUN npm run-script build
 # Expose service on port 3000.
 EXPOSE 3000
 
-# Start service.
-CMD [ "npm", "start" ]
\ No newline at end of file
+# Specify a command for up to this step.
+CMD [ "npm", "start" ]
+
+# Multi-staged build, we don't need a full node image to run the app.
+FROM node:alpine as prod
+
+# No need to run as root.
+USER node
+
+# Run with node env set to production.
+ENV NODE_ENV production
+
+WORKDIR /usr/src/app/
+
+# Copy compiled app from previous stage.
+COPY --from=dev /usr/src/app/data-serving/data-service/node_modules ./node_modules
```

Comment:
What's up with copying the code to a different directory? Is it to avoid tmp file collisions?

Reply:
there's no need to run the app from this long-named-folder-that-was-like-this-due-to-build-context, so in prod the app just runs from /usr/src/app/